anisoap/representations/radial_basis.py

```diff
@@ -27,6 +33,12 @@ def __init__(self, radial_basis, max_angular, **hypers):
             num_n = (max_angular - l) // 2 + 1
             self.num_radial_functions.append(num_n)
 
+        # As part of the initialization, compute the orthonormalization matrix for GTOs
+        # If we are using the monomial basis, set self.overlap_matrix equal to None
+        self.gto_overlap_matrix = None
+        if self.radial_basis == "gto":
+            self.gto_overlap_matrix = self.calc_gto_overlap_matrix()
+
```

Comment:
Small API thing -- let's call this `overlap_matrix`, so that if we're working with non-gto, we don't have a variable named something about gto's